### DISCUSSION TITLE
Are we still updating new jobs?

### DISCUSSION THREAD

**Discussion by Specter43** [ReaVNaiL/New-Grad-2024#24] [Q&A]
There are some more on the market now, but no one updated?

**Comment by uiwree** [comment]
Where did you find them? ☺️

**Reply by Specter43** [reply]
BAE SYSTEM,Pacific, Verizon, JP Morgan, Nutanix...

**Reply by bryanipablo** [reply]
Did you find these through job boards or their career page?

**Reply by Specter43** [reply]
Some are from their website, some are from job boards.

**Reply by ayadav16** [reply]
I just checked for JP Morgan I don't see any open position for a new grad (technology, full-time, Americas as filter criteria)

**Reply by AllenDia** [reply]
What company is "Pacific"?

**Reply by uiwree** [reply]
Appreciated!

**Comment by sjo-time** [comment]
https://careers.jpmorgan.com/us/en/students/programs/software-engineer-fulltime?search=&tags=location__Americas__UnitedStatesofAmerica

**Reply by Roshan2534** [reply]
The United States location is not open yet

**Comment by haozihong** [comment]
This board is collaboratively built by people hunting for jobs in this season, so maybe submit a pull request if you find any new opening? Appreciate it.😊

**Comment by ajohnson114** [comment]
Is there a way to make the links such that when we click on the job it takes us to the job posting? For instance, when I click on meta I get redirected back to this repo. Is it just a me thing?

**Reply by ShubhamGupta19** [reply]
It also mentions "No longer available "in that field.
So it will redirect you to the same page because it is either closed or
filled.
